tests/Application/templates/bundles/SyliusShopBundle/Product/Show/Tabs/Details/_description.html.twig

```diff
@@ -1,5 +0,0 @@
-{% if product.description is not empty %}
-    {{ product.description | monsieurbiz_richeditor_render_field }}
-{% else %}
-    {{ 'sylius.ui.no_description'|trans }}.
-{% endif %}
```

Comment:
Why did you remove this file?

Reply:
It was removed automatically when I created the symlink for Application/templates

Reply:
Yes.
But we need them back ^^.